dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/EPersonRestRepository.java

```diff
@@ -54,7 +54,7 @@ public EPersonRestRepository(EPersonService dsoService) {
 
     @Override
     protected EPersonRest createAndReturn(Context context)
-            throws AuthorizeException {
+        throws AuthorizeException {
```

Comment:
there are only formatting change in this class, can you revert them please?

Reply:
I agree these seem like non-standard formatting changes. It'd be best to just revert all changes to `EPersonRestRepository`.  Could that be done quickly @jonas-atmire  or @MarieVerdonck ?